dnit/infrastructure.ts

```diff
@@ -12,11 +12,11 @@ export interface InfraTasks extends GroupsTasksObject {
 };
 
 /** Make tasks needed for operating typescript/terraform infrastructure */
-export async function makeInfraTasks(): Promise<InfraTasks> {
+export async function makeInfraTasks(ageThresholdMs: number = 5 * 60 * 1000): Promise<InfraTasks> {
```

Comment:
Perhaps make this take a  params struct, to enable additional customizations going forward?

@paul-thompson-helix  ?

Reply:
Agree - avoid positional arguments wherever possible.